script.js

```diff
@@ -2,10 +2,51 @@ let navToggle = document.getElementsByClassName("navToggle")[0]
 let navLinks = document.getElementsByClassName("navLinks")[0]
 
 navToggle.addEventListener("click", menuDown);
-function menuDown(){
-    if(navLinks.style.display === "flex"){
-        navLinks.style.display="none"
-    }else{
-        navLinks.style.display="flex"
+function menuDown() {
+    if (navLinks.style.display === "flex") {
+        navLinks.style.display = "none"
+    } else {
+        navLinks.style.display = "flex"
     }
 }
+
+let navbar = document.getElementsByClassName("navbar")[0]
+
+window.onscroll = function () {
+    "use strict";
```

Comment:
I guess you can use strict mode globally on the file by moving this line to the top. Or possibly you have some reason for limiting it to this function scope?